v1-preparations/LIST_OF_STABLE_RESOURCES_FOR_V1.md

```diff
@@ -0,0 +1,107 @@
+We estimate the given list to be accurate, but it may be subject to small changes:
+
+* Account (in progress)
+    * [snowflake_account](https://registry.terraform.io/providers/Snowflake-Labs/snowflake/0.97.0/docs/resources/account)
+    * [snowflake_accounts](https://registry.terraform.io/providers/Snowflake-Labs/snowflake/0.97.0/docs/data-sources/accounts)
+* Connection (in progress)
+    * snowflake_connection
+    * snowflake_connections
```

Comment:
This and `snowflake_accounts` should be marked as data sources.

Reply:
Done.